queries.sql

```diff
@@ -68,3 +70,25 @@ WHERE date_of_birth
 BETWEEN '1990-01-01' 
 AND '2000-12-31' 
 GROUP BY species;
+
+-- Project 3
+--Join Query by owner name
+select * from animals JOIN owners on animals.owner_id=owners.id where owners.full_name='Melody Pond';
+
+--Join Query by species name
+select * from animals JOIN species on animals.species_id=species.id where species.name='Pokemon';
+
+--Full Join Query by owner
+select * from animals Full JOIN owners on animals.owner_id = owners.id;
+
+--Join Count group by species
+select species.name as animals_species,count(species.name) as species_count from species JOIN animals on animals.species_id=species.id GROUP BY species.name;
+
+--Join three tables and filter by owner and species
+select * from animals JOIN owners on animals.owner_id=owners.id JOIN species on animals.species_id=species.id where species.name='Digimon' and owners.full_name='Jennifer Orwell';
+
+--Join animal and owner table and filter by owner name and zero escape attempt
+select * from animals JOIN owners on animals.owner_id=owners.id WHERE owners.full_name='Dean Winchester' AND escape_attempts=0;
+
+-- Who owns the most animals;
+select full_name as owner_name, count(full_name)no_of_animals from owners JOIN animals on animals.owner_id=owners.id GROUP BY owners.full_name;
```

Comment:
- [ ] I couldn't find the correct answer for this queries, please 🙏🏽 correct the print screens and queries to have the correct information of the requirements. 👇🏽 

`Sam Smith` 1
`Jennifer Orwell` 2
`Bob` 2
`Melody Pond` 3
`Dean Winchester` 2

List all Digimon owned by Jennifer Orwell. = `Gabumon`
List all animals owned by Dean Winchester that haven't tried to escape.
Who owns the most animals? = `Melody Pond` 3

Reply:
some data were inserted twice that's the reason some output data where altered
![Screenshot (56)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/9a7168ff-621f-431e-bc3b-88de5e3618df)

![Screenshot (57)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/9c0950c1-c1ff-44c0-978a-06b6a8862e8b)

some data were inserted twice that's the reason some output data where altered

![Screenshot (59)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/925e2c3f-4888-481d-85d4-0e0bc013f808)